src/create_disk.sh

```diff
@@ -256,44 +256,38 @@ fi
 # be overridden below
 bootloader_backend=none
 
+# Install UEFI on supported architectures
 case "$arch" in
-x86_64)
-    # install bios grub
-    grub2-install \
-    --target i386-pc \
-    --boot-directory rootfs/boot \
-    $disk
+x86_64|aarch64)
+    mkdir -p rootfs/boot/efi/EFI/{BOOT,fedora}
+    mkdir -p rootfs/boot/grub2
     ext="X64"
-
-	# install uefi grub
-	mkdir -p rootfs/boot/efi/EFI/{BOOT,fedora}
+    if [ "${arch}" = aarch64 ]; then
+        ext="AA64"
+    fi
 	cp "/boot/efi/EFI/BOOT/BOOT${ext}.EFI" "rootfs/boot/efi/EFI/BOOT/BOOT${ext}.EFI"
```

Comment:
Minor: not new, though looks like indentation is off here?

Reply:
We have a big mix of tabs and spaces...

Reply:
Ahh yup, classic.